applications/nrf5340_audio/src/bluetooth/le_audio_bis_headset.c

```diff
@@ -135,9 +147,16 @@ static struct bt_audio_stream_ops stream_ops = { .started = stream_started_cb,
 static bool scan_recv_cb(const struct bt_le_scan_recv_info *info, struct net_buf_simple *ad,
 			 uint32_t broadcast_id)
 {
-	LOG_DBG("Broadcast source found, waiting for PA sync");
+	char name[DEVICE_NAME_PEER_LEN];
 
-	return true;
+	bt_data_parse(ad, adv_data_parse, (void *)name);
+
+	if (strcmp(name, DEVICE_NAME_PEER) == 0) {
```

Comment:
If would be better for also checking the DEVICE_NAME_PEER_LEN.

Reply:
I dont think this is needed since 'name' will always be DEVICE_NAME_PEER_LEN and DEVICE_NAME_PEER will always be DEVICE_NAME_PEER_LEN

Reply:
I got it now after a call with Rick:
The problem is if e.g. I have a broadcaster named NRF5340_AUDIO and someone else have another one named NRF5340_AUDIO_TEST

Reply:
Where the first part of the name is similar